### PR TITLE
webpack builder: set bail to false

### PR DESCRIPTION
**Pull request by echarles**
## References

Fixes: https://github.com/jupyterlab/jupyterlab/issues/9435

## Code changes

Set `bail` to `false` in the `webpack.config` of the `builder` package.

## User-facing changes

None

## Backwards-incompatible changes

None

### CODE REVIEW

File: builder/src/webpack.config.base.ts [hunk 1]
```diff
@@ -73,7 +73,7 @@ try {
 }
 
 module.exports = {
-  bail: true,
+  bail: false,
```
Comment (jasongrout) [inline builder/src/webpack.config.base.ts:76]: Since this is the default value, it would be better just to remove it.

**Comment by jasongrout** [comment]
`git log -S "bail: true"` gives 6e7d4bfee2134fa1f3efd3fcd5b271d61cac5161 as the first commit where it appears

**Comment by jasongrout** [comment]
> So the question I think is - do we ever want the bail option set? For example, in the ci tests?

Our understanding was that the proposed fix at #9435 was to only eliminate the `bail` option when watching. Can you do that, rather than eliminating `bail` entirely?

**Comment by echarles** [comment]
Thx for the review. I have update the fix to not bail only in watch mode

```js
const watch = process.argv.includes('--watch');

module.exports = {	module.exports = {
  bail: true,	  bail:  !watch,
```

**Review by blink1073 — APPROVED** [review]
Thanks!  This needs a lint pass but otherwise looks good to me.

**Comment by echarles** [comment]
Thx @blink1073 I have linted and merged.

**Pull request merged**